### ISSUE TITLE
trianer develop using Ignite

### ISSUE THREAD

**Issue by Ldoun** [Sing-it/DeepSinger#2]
**Implement**
- TBTT algorithm
- Greedy training
- Guided attention loss

**Comment by Ldoun** [comment]
val loss doesn't decrease -> so decided to develop using pytorch Lightning
